src/tye/CleanupHost.cs

```diff
@@ -0,0 +1,220 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using k8s;
+using k8s.Models;
+using Microsoft.Rest;
+using Microsoft.Tye.ConfigModel;
+using Newtonsoft.Json;
+
+namespace Microsoft.Tye
+{
+    public static class CleanupHost
+    {
+        public static async Task CleanupAsync(IConsole console, FileInfo path, Verbosity verbosity, bool interactive, bool whatIf)
+        {
+            var output = new OutputContext(console, verbosity);
+
+            output.WriteInfoLine("Loading Application Details...");
+
+            // We don't need to know anything about the services, just the application name.
+            var application = ConfigFactory.FromFile(path);
+
+            await ExecuteCleanupAsync(output, application, interactive, whatIf);
+        }
+
+        public static async Task ExecuteCleanupAsync(OutputContext output, ConfigApplication application, bool interactive, bool whatIf)
+        {
+            var config = KubernetesClientConfiguration.BuildDefaultConfig();
```

Comment:
Would we want this cleanup task to be reusable in anyway (or pluggable into the deploy pipeline)? I think this is fine for now though to keep it simple.

Reply:
I want to keep it simple. We can refactor when needed.